main.go

```diff
@@ -152,7 +149,14 @@ func main() {
 
 				// Create cache
 				cache := job.NewMemoryJobCache(db)
-				cache.Start(time.Duration(c.Int("persist-every")) * time.Second)
+
+				duration, err := time.ParseDuration(c.String("persist-every"))
+				// if duration string could not be parsed, set duration to default
+				if err != nil {
+					cache.Start(time.Duration(5 * time.Minute))
```

Comment:
I think we have two options:
- Let the program crash if the duration string could not be parsed and show an error.
- Show a message that the duration string could not be parsed and that the persistant interval was set to the default value.